scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -17,6 +17,13 @@ workflow GvsExtractCallset {
         String? fq_filter_set_site_table
         String? fq_filter_set_tranches_table
         String? filter_set_name
+
+        # if these are unset, default sensitivity levels will be used; if user is defining a cutoff, only use one type (sensitivity or lod)
+        Float? snps_truth_sensitivity_filter_level
```

Comment:
Since providing nothing uses a default (and is the common case) I wonder if these should have a name more like "snps_truth_sensitivity_filter_level_override" or something to make it clear saying nothing is ok?

Reply:
great idea